jina/peapods/__init__.py

```diff
@@ -32,7 +33,7 @@ def Pea(args: 'argparse.Namespace' = None, allow_remote: bool = True, **kwargs):
     if args.host != __default_host__:
         from .remote import RemotePea
         return RemotePea(args)
-    elif args.image:
+    elif args.uses and valid_docker_image(args.uses):
         from .container import ContainerPea
         return ContainerPea(args)
```

Comment:
```
elif args.uses and not args.uses.endwith(('.yml', 'yaml')):
        if valid_docker_image(args.uses):
                from .container import ContainerPea
                return ContainerPea(args)
        else:
                print(f'No docker image named {args.uses}.')
```

Reply:
Valid yaml paths are not the only alternative to docker images for `uses`. They can also be some keywords like `_forwards`, `_route`, `_merge` or some valid python class names, like `BaseExecutor`.

Reply:
checking the validity of image online may take some time, how can we quickly tell if it's not a yaml?

Reply:
a **try catch** over **valid_yaml_path**, but I find a little ugly! I will try to split **valid_yaml_path** and **get_valid_yaml_path**. **valid_yaml_path** would return a **boolean**, while **get_valid_yaml_path** would return a **readable** object.

Reply:
that will be great